shell/platform/linux/fl_view.cc

```diff
@@ -136,10 +136,15 @@ static gboolean run_flutter_engine(FlView* self) {
   config.open_gl.fbo_callback = fl_view_gl_fbo_callback;
   config.open_gl.present = fl_view_gl_present;
 
+  g_autofree gchar* assets_path = g_build_filename(
+      fl_dart_project_get_path(self->flutter_project), "flutter_assets", NULL);
```

Comment:
I'd like this logic to be par of FlDartProject, rather than the embedding, so that the logic of how a project bundle is structured is encapsulated within the project class.